php/php-xhprof/Portfile

```diff
@@ -1,47 +1,52 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               github 1.0
```

Comment:
This portgroup is not being used. (You haven't called `github.setup`.)
```suggestion
```

Reply:
Applied here, will be part of the next push. Thanks!